### PR TITLE
Switch Zendesk to use encrypted logging

### PR DESCRIPTION
**Pull request by jkmassel**
**To test:**
- Send a test ticket to ZenDesk
- Ensure that it contains an encrypted log UUID
- Ensure that the encrypted log can be fetched from `mc`

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/Utility/ZendeskUtils.swift [hunk 1]
```diff
@@ -611,21 +612,33 @@ private extension ZendeskUtils {
...
-                return ""
+        let fileLogger = WPLogger.shared().fileLogger
+        let dataProvider = EventLoggingDataProvider.fromDDFileLogger(fileLogger)
+
+        guard let logFilePath = dataProvider.logFilePath(forErrorLevel: .debug, at: Date()) else {
+            return "Error: No log files found on device"
         }
 
-        // Truncate the log text so it fits in the ticket field.
-        if logText.count > Constants.logFieldCharacterLimit {
```
Comment (ScoutHarris) [inline WordPress/Classes/Utility/ZendeskUtils.swift:624]: Since `Constants.logFieldCharacterLimit` is no longer being used, it can be removed.

**Review by ScoutHarris — APPROVED** [review]
I was able to view my logs via mc. One minor nitpick, otherwise :shipit: .

**Review by designsimply — APPROVED** [review]
Tested and besides me not realizing that the UUID is stored in a field named System Status Report at first 🙂 it works as described. 

<sup>(internal reference: 3558120-zen)</sup>

File: WordPress/Classes/Utility/ZendeskUtils.swift (hunk 1 above)
Comment (jkmassel) [reply WordPress/Classes/Utility/ZendeskUtils.swift:624]: Ooh nice catch. Thank you!!

**Pull request merged**